chatmeleon-server/utils/generate-test-record.js

```diff
@@ -0,0 +1,60 @@
+import { PrismaClient } from '@prisma/client';
```

Comment:
This needs comments to show its purposes or in which context it will be used at the beginning of the file

Reply:
What's the purpose of this file?